articles/active-directory/conditional-access/howto-baseline-protect-administrators.md

```diff
@@ -69,6 +69,9 @@ To enable this policy and protect your administrators:
 1. Add any user exclusions by clicking on **Users** > **Select excluded users** and choosing the users that need to be excluded. Click **Select** then **Done**.
 1. Click **Save**.
 
+> [!WARNING]
+> There was an option **Automatically enable policy in the future** when this policy was in preview. We removed this option to minimize sudden user impact. If you selected this option when it was available, **Do not use policy** is automatically now selected. If they want to use this baseline policy, see steps above to enalbe it.
```

Comment:
Ill make some edits to this when I do my next update to the doc but looks fine for now. Use of multiple tense You/They and should just be a note not a warning.